js/file_load_worker.js

```diff
@@ -142,22 +158,43 @@ async function loadBugFile(bugFile, list) {
             list.push({
```

Comment:
Do we really want to maintain the old legacy format for capturing a view hierarchy? What if we just stopped supporting it since the user base is so small and all the customers are developers on SysUI?